### DISCUSSION TITLE
Link from pages and footer to a versioned doc

### DISCUSSION THREAD

**Discussion by lorenzolewis** [facebook/docusaurus#7397] [Q&A]
We are currently using the second method discussed in https://docusaurus.io/docs/versioning#configuring-versioning-behavior so that our "latest" docs are served under a version number (v1 in this scenario). 

For navBar items, you can use `type: 'doc'` to dynamically build the url to the correct doc version. I'm not able to recreate this behaviour for footer items linking to docs. I'm also not seeing how to link to a doc from a page (such as `index.js`) to the correct "latest" doc url.

**Comment by Josh-Cena** [comment]
Let's discuss in #7402 instead
